### ISSUE TITLE
I got very low mAP...

### ISSUE THREAD

**Issue by OrchidBrando** [layumi/Person-reID-triplet-loss#20]
Hello, I used your code to train 499 epochs with "python train_new.py --gpu_ids 0 --name ft_ResNet50 --train_all --batchsize 16 --data_dir ./Market". I didn’t change the parameters of the code, but the results were not great. The final result is: Rank@1:0.803147 Rank@5:0.917755 Rank@10:0.944181 mAP:0.614890
Is this because I did not use "--erasing_p 0.5"?

**Comment by layumi** [comment]
@OrchidBrando 

It may be the reason. More epochs sometime also can help the performance. 

You may consider to add cross-entropy loss for helping the model learning.
